mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py

```diff
@@ -151,8 +42,12 @@ class mgmtsystem_nonconformity_severity(orm.Model):
 ]
 _STATES_DICT = dict(_STATES)
 
+own_company = lambda self: self.env.user.company_id.id
+default_date = lambda *a: time.strftime(DATE_FORMAT)
+default_user_id = lambda self: self.env.user.id
```

Comment:
It shouldn't but the travis tests did pass. There might be something wrong with the tests.